components/css/componentStyle/Image.css

```diff
@@ -0,0 +1,19 @@
+.round__image-div{
+    height: 15rem;
+    width: 15rem;
+    border-radius: 50%;
+    margin: 0rem 2rem 0rem 2rem;
+}
+
+.round__img{
+    width: 100%;
+    height: 100%;
+    border-radius: 50%;
+    object-fit: cover;
```

Comment:
You have already mentioned `object-fit` in `img_style`. Don't repeat.  Also, change class name to `img__round`